portal/src/main/scala/cromwell/pipeline/datastorage/dao/entry/UserEntry.scala

```diff
@@ -31,4 +31,12 @@ class UserEntry(val pipelineDatabaseEngine: PipelineDatabaseEngine) {
 
   def addUserAction(user: User) = (users.returning(users.map(_.userId))) += user
 
+  def updateUser(updatedUser: User) =
+    users
+      .filter(_.userId === updatedUser.userId)
+      .map(user => (user.email, user.firstName, user.lastName))
+      .update((updatedUser.email, updatedUser.firstName, updatedUser.lastName))
+
+  def updateUserPassword(updatedUser: User) =
+    users.filter(_.userId === updatedUser.userId).map(user => (user.passwordHash)).update((updatedUser.passwordHash))
```

Comment:
You don't have to work with `User` entity here.
You're working with `UserUpdateRequest` and that's great! So pass it here.
If you do this, you won't be forced to redefine `User` entity with empty lines or do other not-nice stuff.